zipline/sources/benchmark_source.py

```diff
@@ -29,7 +31,9 @@ def __init__(self, benchmark_sid, env, trading_days, data_portal,
         self.emission_rate = emission_rate
         self.data_portal = data_portal
 
-        if self.benchmark_sid:
+        if len(trading_days) == 0:
+            self._precalculated_series = pd.Series()
```

Comment:
I noticed that we're not setting the `self.benchmark_asset` attribute in this case.  Should we set it to a default value?  Started to look at who uses it, and I'm confused by `_initialize_precalculated_series` getting passed `self.benchmark_asset` but also reading it directly.  Maybe we don't need a default value, but should pass it around in general?